doc/index.rst

```diff
@@ -1,17 +1,17 @@
-***********************************************
+********************************************
 Welcome to the NEST simulator documentation!
-***********************************************
+********************************************
 
 :orange:`Introducing NEST 3.0!`
 -------------------------------
 
 Our aim with NEST 3.0 is to provide a more intuitive experience with simplified yet versatile handling and manipulation of nodes and connections.
 
 
-- We have provided a :doc:`guide to NEST 3.0 <guides/from_nest2_to_nest3>` , including examples of the new functionality.
+- We have provided an :doc:`overview of NEST 3.0 <guides/nest2_to_nest3/nest2_to_nest3_overview>`, including examples of the new functionality.
```

Comment:
How about: "Get an :doc:`overview of NEST 3.0 ...`, including ..."? And should we capitalise the name of the guide as a title, i.e., "Overview of NEST 3.0".

Reply:
I agree about making the sentence more direct and capitalizing the title. 

How about ``Visit our What’s new? guide to get an overview of NEST 3.0 and the new functionality``? 

Since ``What's new?`` is the name of the guide.

Reply:
Good!